src/main/java/com/mentorme/mentor/controller/EventController.java

```
@@ -1,27 +1,38 @@
 package com.mentorme.mentor.controller;
 
-import com.mentorme.mentor.dto.UserDto;
-import com.mentorme.mentor.service.user.UserService;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
-
+import com.mentorme.mentor.dto.Event.EventDto;
+import com.mentorme.mentor.dto.Event.NewEventDto;
+import com.mentorme.mentor.service.event.EventService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
 
 @RestController
+@RequestMapping(path = "/event")
 public class EventController {
-    private UserService userService;
-
-    public  EventController(UserService userService){ this.userService = userService; }
-
-    @RequestMapping(path = "add", method = RequestMethod.GET)
-    public UserDto addNewUser(){
-        String name = "dan";
-        Integer roleId  = 10;
 
+    @Autowired
+    private EventService eventService;
+
+    @RequestMapping(value = "/add/{roleId}", method = RequestMethod.POST)
+    public EventDto addUser(@PathVariable Integer roleId,
+                               @RequestBody NewEventDto newEventDto) {
+        if (roleId >= 2) {
+            return eventService.save(newEventDto);
+        } else {
+            return null;
```

It is a good practice to throw errors instead of returning nulls. Write a generic exception for the invalid request scenario. It should be an unchecked exception. Then, throw it from here.